CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.11.1
```

Comment:
Best to add heading `UNRELEASED` since it's not known how many features may be added prior to the next release. For future reference, as I can just modify the heading when ready to publish.